.github/workflows/lint.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [18.x]
```

Comment:
Let's please stop using matrix when it's not needed

Reply:
removed in the next PR 👍